server/src/main/java/com/exacaster/lighter/rest/MicronautStatementCreationResultToResponseMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.exacaster.lighter.rest;
+
+import com.exacaster.lighter.application.ApplicationState;
+import com.exacaster.lighter.application.sessions.Statement;
+import com.exacaster.lighter.application.sessions.StatementCreationResultMapper;
+import io.micronaut.http.HttpResponse;
+
+public class MicronautStatementCreationResultToResponseMapper implements StatementCreationResultMapper<HttpResponse> {
```

Comment:
It could be just `StatementCreationResultToApiResponseMapper`

Reply:
can rename it, but pls remember that this mapper is micronaut specific. 
If you decide to change web layer or expose functionality via different medium like messaging, this implementation needs to be changed

Reply:
Sure, the result value `HttpResponse` kind of makes it visible, that it is Micronaut specific.